src/components/RecipeFind.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <div id="recipe-find">
-    <b-form-input @keydown.enter="onEnter" v-model.trim="search"></b-form-input>
+    <b-form-input @keydown.enter="onEnter" v-on:keydown="onKeydown" v-model.trim="search">
```

Comment:
`onKeyDown` selects the first item in the list. So, this will need to be updated to select the highlighted option.

Reply:
This is also done by adding and removing "highlight" class based on mouse position. @alfg